### ISSUE TITLE
Custom CSS doesn't work

### ISSUE THREAD

**Issue by aluminum1** [typora/typora-issues#2884]
Thanks for a great program. 

I'm using Typora Version 0.9.9.28.5 (3442) on Mac and the custom CSS doesn't work. 
For instance, I make a file base.user.css inside the themes folder, with the following:

`body {
  font-family: Courier;
}
`

But the font doesn't change in the file I'm editing in Typora - even if I save and exit Typora and then open Typora again. However, if I export to HTML, then the CSS code is included in the resulting HTML file.

**Comment by aluminum1** [comment]
Works after I save, close, reload.

**Comment by codewizard13** [comment]
I was experiencing this exact same problem in April of 2021. I am on Windows 10, but I discovered the solution and _**I'm posting for others who might be in a similar circumstance and might benefit from this info**_ For me, what I had done was grabbed this CSS file from the Typora repo:

https://raw.githubusercontent.com/typora/typora-theme-toolkit/master/template.less

I renamed it **base.user.css** as the directions here (https://support.typora.io/Add-Custom-CSS/) indicated and then changed background to yellow and text to green. Unfortunately, even after closing and restarting there was no change. I troubleshooted the issue for about 20 minutes when I finally figured it out:

This is a LESS template with less variables. It needs to be compiled to a CSS file to take effect!

Here are the steps I took to fix the issue:
1) Close all Typora instances
1) Renamed base.user.css to _**base.user.less**_
2) Opened the less file in Brackets and prepended this line:
>// out: base.user.css
4) Saved file _(LESS AutoCompile plugin must be installed in Brackets_) auto-generates the ".css" file in same folder
5) Opened Typora --> Voila! The background was yellow so the css file was definitely being imported.
